src/components/home/home.scss

```diff
@@ -1,294 +1,95 @@
-/******************************
-    HERO BLOCK
-*******************************/
-$bg-img: url('../../images/meal-prep.jpg');
-$teal-empahasis-color: #01B4BC;
-$red-cta-color: #FA5457;
-$white-cta-color: #ffffffd8;
-$cta-btn-font: 'Muli', sans-serif;
 
-#hero-block {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  background: $bg-img no-repeat center center;
-  background-size: cover;
+section {
+  min-height: 100vh;
   width: 100%;
-  height: 100%;
-
-  .cta-block {
-    box-sizing: border-box;
-    background-color: $white-cta-color;
-    text-align: center;
-    height: 100%;
-    width: 100%;
-    padding: 10%;
-      h1 {
-        padding-top:50px;
-        font-size: 1.8rem;
-        font-weight:300;
-      }
-
-      p {
-        font-size: 1.1rem;
-      }
-
-      .cta-btn {
-        background-color: $red-cta-color;
-        width: 100%;
-        color: white;
-        font-weight: 100;
-        font-size: 1rem;
-        font-family: $cta-btn-font;
-        text-transform: uppercase;
-        padding: 15px;
-        border: 1px solid #000;
-        margin-bottom: 70px;
-      }
-  }
+  display: flex;
 }
 
-@media only screen and (min-width: 700px) {
+/*      MEDIA QUERIES     */
+/* small phones to small tablets: from 481px to 767px */
+@media only screen and (max-width: 767px) {
```

Comment:
With mobile-first development, min-width is typically used rather than max-width. SSo the organization/logic would be backwards in nesting and might look like this for scss

`
# Smallest up to the first breakpoint. Let's say, 700px
section {
     flex-direction: column;
     
     # screen sizes starting from 700px and above
     @media only screen and (min-width: 700px) {
            min-height: 100vh;
            width: 100%;
            display: flex;
      }
}
`